torch/linalg/__init__.py

```diff
@@ -339,6 +339,88 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+cond = _add_docstr(_linalg.linalg_cond, r"""
+linalg.cond(input, p=None, *, out=None) -> Tensor
+
+Computes the condition number of a matrix :attr:`input`,
+or of each matrix in a batched :attr:`input`, using the matrix norm defined by :attr:`p`.
+The condition number is defined as the matrix norm of
```

Comment:
Should this be something like,

"For norms ... this is defined as the matrix norm of ... And for norms ... this is defined as the ratio ..."

per the latest update?

Reply:
Yes, I've changed that.